manifests/stage-ipoib-cni/0010-ipoib-cni-ds.yml

```diff
@@ -0,0 +1,72 @@
+# 2022 NVIDIA CORPORATION & AFFILIATES
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kube-ipoib-cni-ds
+  namespace: {{ .RuntimeSpec.Namespace }}
+  labels:
+    tier: node
+    app: ipoib-cni
+    name: ipoib-cni
+spec:
+  selector:
+    matchLabels:
+      name: ipoib-cni
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        tier: node
+        app: ipoib-cni
+        name: ipoib-cni
+    spec:
+      hostNetwork: true
+      affinity:
+        nodeAffinity:
+        {{- if not .NodeAffinity }}
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
```

Comment:
please update like the other daemonsets to account for the control-plane label

i.e add

```
                  - key: "node-role.kubernetes.io/control-plane"
                    operator: DoesNotExist
```